src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -31,36 +72,36 @@ iframe {
 }
 a:link, a:visited {
     text-decoration:none;
-    color:#4A6782;
+    color:var(--link-color);
 }
 a[href]:hover, a[href]:focus {
     text-decoration:none;
-    color:#bb7a2a;
+    color:var(--link-color-active);
 }
 a[name] {
     color:#353833;
 }
 pre {
-    font-family:'DejaVu Sans Mono', monospace;
-    font-size:14px;
+    font-family:var(--code-font-family);
+    font-size:1em;
 }
 h1 {
-    font-size:20px;
+    font-size:1.428em;
```

Comment:
I appreciate these numbers give per-pixel consistency with before, but 3 decimal places seems a bit extreme!

Reply:
I agree. I only went to the third decimal position when it was actually necessary to retain the exact font size. But it would probably not have caught anybody's eye had I rounded to the nearest two decimal number. I guess I could do that to make it more uniform.

Reply:
I would leave it for now, for this release, and maybe we tweak/simplify the constants later on.